README.md

```diff
@@ -1,5 +1,9 @@
 # cfgov-refresh
 
+[![Build Status](https://travis-ci.org/cfpb/cfgov-refresh.png?branch=flapjack)](https://travis-ci.org/cfpb/cfgov-refresh?branch=flapjack)
+[![Coverage Status](https://coveralls.io/repos/cfpb/cfgov-refresh/badge.png?branch=flapjack)](https://coveralls.io/r/cfpb/cfgov-refresh?branch=flapjack)
```

Comment:
Probably should remove coveralls badge for now till coveralls can pull in a different branch than master. Code Climate also has a coverage badge that comes up with unknown. If we can use the Code Climate one then it reduces the services.